res/database/Default/user/scr0llbaer_constr_d2v_tv_series.xml

```diff
@@ -0,0 +1,6397 @@
+			</title>
+			<title_original>
+				<de>Wallace &amp; Gromit - Die Techno-Hose</de>
+				<en>Wallace and Gromit: The Wrong Trousers</en>
+			</title_original>
+			<description>
+				<de>Der Erfinder hat ein besonderes Geschenk zum Geburtstag seines Hundes: Roboterhosen, die sich von selbst bewegen können. Außerdem vermietet er ein Zimmer im Haus an einen zwielichten Pinguin.</de>
+				<en>The inventor has a special gift for his dog's birthday: robot pants that can move by themselves. He also rents out a room in the house to a shady penguin.</en>
+			</description>
+			<staff>
+				<member index="0" function="5">f24c9932-a071-47ef-a650-6856bcb0153c</member>
+				<member index="1" function="2">various-voice-artists</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" />
+			<targetgroupattractivity teenagers="1.5" housewives="1.5" employees="1.5" unemployed="1.25" managers="1.5" pensioners="2" />
+			<data country="GB" distribution="2" maingenre="3" subgenre="5,1" flags="10" blocks="1" price_mod="0.5" />
+			<releaseTime year="1993" day="12" />
+			<ratings critics="88" speed="84" outcome="83" />
+			<modifiers>
+				<modifier name="topicality::age" value="0.5" />
+			</modifiers>
+		</programme>
+		<programme id="d87c78b8-b55b-40c6-916c-5f2e45630736" product="1" licence_type="1" creator="9551" imdb_id="tt0112691">
+			<title>
+				<de>Wally &amp; Grombit - Eine gründliche Rasur</de>
+				<en>Wally &amp; Grombit: Among Sheep</en>
+			</title>
+			<title_original>
+				<de>Wallace &amp; Gromit - Unter Schafen</de>
+				<en>Wallace and Gromit: A Close Shave</en>
+			</title_original>
+			<description>
+				<de>Der Erfinder verliebt sich in Wendolene, doch die steht mit der örtlichen Wollknappheit in Verbindung, was dazu führt, dass er wegen Schafsdiebstahl inhaftiert wird. Mit einem Wollpullover tragenden Lamm plant er den Ausbruch.</de>
+				<en>The inventor falls in love with Wendolene, but she has ties to the local wool shortage, which leads to his imprisonment for sheep-rustling. Together with a woolly jumper-wearing lamb, he plans to escape.</en>
+			</description>
+			<staff>
+				<member index="0" function="5">f24c9932-a071-47ef-a650-6856bcb0153c</member>
+				<member index="1" function="2">various-voice-artists</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" />
+			<targetgroupattractivity teenagers="1.5" housewives="1.5" employees="1.5" unemployed="1.25" managers="1.5" pensioners="2" />
+			<data country="GB" distribution="2" maingenre="3" subgenre="5,1" flags="10" blocks="1" price_mod="0.5" />
+			<releaseTime year="1995" day="12" />
+			<ratings critics="86" speed="84" outcome="81" />
+			<modifiers>
+				<modifier name="topicality::age" value="0.5" />
+			</modifiers>
+		</programme>
+		<programme id="0fa02d35-1e99-4f77-8806-43f31eeb5a8e" product="1" licence_type="1" creator="9551" imdb_id="tt0109552">
+			<title>
+				<de>Darkguy II - Durant ist zurück</de>
```

Comment:
First part is in the other file because it was a cinema movie, part 2 and 3 are direct-to-video. For my internal "book-keeping" I keep these separate.